### PR TITLE
Develop to Master: Master version 2.0

### PR DESCRIPTION
**Pull request by jmfadeley**

### CODE REVIEW

**Comment by jsachs** [comment]
Any chance of getting the linting PR into this?

Also woo for no merge conflicts (for now).

**Comment by jmfadeley** [comment]
Sure. Give me a few.

**Comment by jmfadeley** [comment]
@jsachs Done.

File: .env [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SURVEY_SERVICE_DB_NAME=survey
+SURVEY_SERVICE_DB_PORT=5432
+SURVEY_SERVICE_DB_USER=amida
+SURVEY_SERVICE_DB_PASS=amida
+SURVEY_SERVICE_DB_DIALECT=postgres
+SURVEY_SERVICE_DB_POOL_MAX=5
+SURVEY_SERVICE_DB_POOL_MIN=0
+SURVEY_SERVICE_DB_POOL_IDLE=10000
+SURVEY_SERVICE_LOGGING_LEVEL=emerg
+SURVEY_SERVICE_CORS_ORIGIN=*
```
Comment (Perry5) [inline .env]: We are missing some env vars here: this is the complete list I have on greyscale:
```
INDABA_PG_USERNAME=
INDABA_PG_TESTUSER=
INDABA_PG_PASSWORD=
INDABA_PG_DB=
INDABA_PG_HOSTNAME=
AUTH_SALT=
JWT_SECRET=
SURVEY_SERVICE_URL=
AUTH_SERVICE_URL=
DOMAIN=
GMAIL_PASS=
MESSAGE_SERVICE_URL=
SYS_MESSAGE_USER=
SYS_MESSAGE_PASSWORD=
S3_BUCKET=
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
MAILER_EMAIL_ID=
MAILER_PASSWORD=
MAILER_SERVICE_PROVIDER=
ADMIN_USER_EMAIL=
ADMIN_USER_FIRST_NAME=
ADMIN_USER_LAST_NAME=
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### What's this PR do?
```
Comment (Perry5) [inline PULL_REQUEST_TEMPLATE.md]: For whatever reason, this PR Template never worked for us on the `develop` branch. Maybe it'll work on master after this merge

File: README.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ Indaba puts your stakeholder and expert network at your fingertips. It converts
 - pgAdmin (optional)
 - memcached
 - nginx (for server deployment)
+- Docker
```
Comment (Perry5) [inline README.md]: I know we want to get this merge in asap but should we maybe update this README with this file:
https://paper.dropbox.com/doc/Indaba-Set-up-for-new-developers-Mac-5NzKPSGxbHOUXomHFaEKy

Comment (jmfadeley) [reply README.md]: @Perry5 I have doubts because that only covers it for Mac. How about we push this as is and revisit that?

Comment (Perry5) [reply README.md]: 👍

**Pull request merged**